core/chains/evm/config/config.go

```diff
@@ -1069,6 +1073,41 @@ func (c *chainScopedConfig) EvmGasTipCapMinimum() *big.Int {
 	return &c.defaultSet.gasTipCapMinimum
 }
 
+// NodeNoNewHeadsThreshold controls how long to wait after receiving no new
+// heads before marking the node as out-of-sync
+// Set to zero to disable out-of-sync checking
+func (c *chainScopedConfig) NodeNoNewHeadsThreshold() time.Duration {
+	val, ok := c.GeneralConfig.GlobalNodeNoNewHeadsThreshold()
+	if ok {
+		c.logEnvOverrideOnce("NodeNoNewHeadsThreshold", val)
+		return val
+	}
+	return c.defaultSet.nodeDeadAfterNoNewHeadersThreshold
```

Comment:
Any reason why all these new entries can't be set via c.persistedCfg, like some configs just above these?

Reply:
Ok, as long as we are planning to revisit and fix these configs, we are good.